src/api/generated/wrapper/account.rs

```diff
@@ -1,11 +1,15 @@
 use std::collections::HashMap;
 
-use crate::api::generated::execution::account as account_execution;
-use crate::api::generated::response::account;
-use crate::api::utils::utils;
-use crate::client::client;
-
-use serde_json;
+use crate::{
```

Comment:
Is this idiomatic, or just a different way of doing it?

Reply:
You mean with collapsing import statements? I would say it's idiomatic, yeah. [Cargo fmt](https://github.com/rust-lang/rustfmt) doesn't support automatically doing it yet, but [it's available if you use nightly rust](https://rust-lang.github.io/rustfmt/?version=master&search=merge#merge_imports).